chatwoot-plugin.php

```diff
@@ -39,30 +39,37 @@ function chatwoot_assets() {
 /**
  * Initialize embed code options.
  *
+ * @link https://developer.wordpress.org/reference/functions/wp_localize_script/
+ *
  * @since 0.1.0
  *
  * @return {void}.
  */
 function chatwoot_load() {
 
-  // Get our site options for site url and token.
-  $chatwoot_url = get_option('chatwootSiteURL');
-  $chatwoot_token = get_option('chatwootSiteToken');
-  $chatwoot_widget_locale = get_option('chatwootWidgetLocale');
-  $chatwoot_widget_type = get_option('chatwootWidgetType');
-  $chatwoot_widget_position = get_option('chatwootWidgetPosition');
-  $chatwoot_launcher_text = get_option('chatwootLauncherText');
+	// Get our site options for site url and token.
+	$chatwoot_url             = get_option( 'chatwootSiteURL' );
+	$chatwoot_token           = get_option( 'chatwootSiteToken' );
+	$chatwoot_widget_locale   = get_option( 'chatwootWidge tLocale' );
```

Comment:
Seems like a typo here

```suggestion
	$chatwoot_widget_locale   = get_option( 'chatwootWidgetLocale' );
```